x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_service.ts

```diff
@@ -65,6 +63,10 @@ export class RuleMigrationsDataService {
         adapterId: 'prebuiltrules',
         fieldMap: prebuiltRulesFieldMap,
       }),
+      migrations: this.createIndexPatternAdapter({
+        adapterId: 'migrations',
+        fieldMap: migrationsFieldMaps,
+      }),
```

Comment:
NIT: Can we move the `migration` index definitions to the top before anything else? Also in the `install`, `createClient` and everywhere. 
So it's evident for other people that this one is the first and most essential document we store for the migration. It's the last one we introduce, but it's the most basic one 😅